src/bot/__tests__/requestReview.test.ts

```diff
@@ -0,0 +1,198 @@
+    const boundShortcutMethod = jest.fn();
+    const boundCallbackMethod = jest.fn();
+
+    beforeEach(() => {
+      app = {
+        shortcut: jest.fn() as any,
+        view: jest.fn() as any,
+      } as App;
+      requestReview.shortcut.bind = jest.fn().mockReturnValueOnce(boundShortcutMethod);
+      requestReview.callback.bind = jest.fn().mockReturnValueOnce(boundCallbackMethod);
+
+      requestReview.setup(app);
+    });
+
+    it('should run shortcut() when the "Request a Review" shortcut is pressed', () => {
+      expect(app.shortcut).toBeCalledWith(Interaction.SHORTCUT_REQUEST_REVIEW, boundShortcutMethod);
+    });
+
+    it('should run callback() after the user submits the "Request a Review" dialog', () => {
+      expect(app.view).toBeCalledWith(Interaction.SUBMIT_REQUEST_REVIEW, boundCallbackMethod);
+    });
+  });
+
+  describe('shortcut', () => {
+    describe('when no errors occur', () => {
+      let param: ShortcutParam;
+
+      beforeEach(async () => {
+        param = buildMockShortcutParam();
+        languageRepo.listAll = jest.fn().mockResolvedValueOnce(['Javascript', 'Go', 'Other']);
+
+        await requestReview.shortcut(param);
+      });
+
+      it("should acknowledge the request so slack knows we're working on it", () => {
+        expect(param.ack).toBeCalled();
+      });
+
+      it("should show a dialog who's submit button triggers the callback() function", () => {
+        expect(param.client.views.open).toBeCalledWith({
+          trigger_id: param.shortcut.trigger_id,
+          view: expect.objectContaining({
+            title: {
+              text: 'Request a Review',
+              type: 'plain_text',
+            },
+            type: 'modal',
+            callback_id: Interaction.SUBMIT_REQUEST_REVIEW,
+            submit: {
+              type: 'plain_text',
+              text: 'Submit',
+            },
+          }),
+        });
+      });
+
+      it('should setup the first response block for the languages used', () => {
+        const { mock } = param.client.views.open as jest.Mock;
+        const blocks = mock.calls[0][0].view.blocks;
+        expect(blocks[0]).toEqual({
+          type: 'input',
+          label: {
+            text: 'What languages were used?',
+            type: 'plain_text',
+          },
+          element: {
+            type: 'checkboxes',
+            action_id: ActionId.LANGUAGE_SELECTIONS,
+            options: [
+              { text: { text: 'Javascript', type: 'plain_text' }, value: 'Javascript' },
+              { text: { text: 'Go', type: 'plain_text' }, value: 'Go' },
+              { text: { text: 'Other', type: 'plain_text' }, value: 'Other' },
+            ],
+          },
+        });
+      });
+
+      it('should setup the second response block for when the reviews are needed by', () => {
+        const { mock } = param.client.views.open as jest.Mock;
+        const blocks = mock.calls[0][0].view.blocks;
+        expect(blocks[1]).toEqual({
```

Comment:
I like how you broke out the assertions for the blocks into separate tests. I think it makes it easier to follow.